src/content/content-ai/capabilities/ai-toolkit/primitives/display-suggestions.mdx

```diff
@@ -227,3 +227,98 @@ function MyComponent() {
   return null
 }
 ```
+
+## `setHTMLSuggestions`
+
+Loads AI-powered suggestions by comparing corrected HTML with the current document content. This method is designed for integration with AI APIs that return corrected text or HTML.
+
+This method supports two input formats:
+
+1. **HTML format**: Provide the corrected HTML and let the toolkit compute the differences automatically
+2. **Changes format**: Provide an array of specific text replacements (insert/delete pairs)
+
+The method uses a diff-based approach to identify changes and creates suggestions at the appropriate document positions.
+
+### Parameters
+
+- `options` (`SetHTMLSuggestionsOptions`): Configuration options
+  - `content?` (`string`): The corrected HTML content from your AI provider. Use this for the HTML-based format. Cannot be used together with `changes`.
+  - `changes?` (`TextChange[]`): Array of text changes in the format `{ insert: string, delete: string }`. Each change specifies text to delete and its replacement. Cannot be used together with `content`.
+  - `range?` (`Range`): Optional range to limit suggestions to a specific portion of the document. If not provided, the entire document will be processed.
+  - `abortSignal?` (`AbortSignal`): Optional abort signal to cancel the operation.
+
+<Callout title="Changes format behavior" variant="info">
+  When using the `changes` format, ALL occurrences of each `delete` text will be replaced with the corresponding `insert` text throughout the specified range. If you need position-specific replacements, use the HTML format instead.
+</Callout>
+
+### Returns
+
+`Promise<SetHTMLSuggestionsResult>`
+
+Returns a promise that resolves to an object containing:
+
+- `suggestions` (`Suggestion[]`): The array of generated suggestions
+
+### Examples
+
+#### HTML format with API integration
+
+```ts
+const toolkit = getAiToolkit(editor)
+
+// Get corrected HTML from your AI API
+const response = await fetch('/api/grammar-check', {
+  method: 'POST',
+  headers: { 'Content-Type': 'application/json' },
+  body: JSON.stringify({ html: editor.getHTML() })
+})
+const { correctedHtml } = await response.json()
+
+// Set suggestions using the corrected HTML
+const result = await toolkit.setHTMLSuggestions({
+  content: correctedHtml
+})
+
+console.log(`Generated ${result.suggestions.length} suggestions`)
+```
+
+#### HTML format with range parameter
+
+```ts
+const toolkit = getAiToolkit(editor)
+
+// Only process a specific range of the document
+await toolkit.setHTMLSuggestions({
+  content: '<p>Corrected paragraph content</p>',
+  range: { from: 0, to: 100 }
+})
+```
+
+#### Changes format with single change
+
+```ts
+const toolkit = getAiToolkit(editor)
+
+// Capitalize all occurrences of 'api' to 'API'
+await toolkit.setHTMLSuggestions({
+  changes: [
+    { insert: 'API', delete: 'api' }
```

Comment:
Nitpick: put the delete before the insert, I believe it's easier to understand this way.